cinch/cinch

```
@@ -11,8 +11,12 @@ USAGE = """{name} [interpret|dump|vm|compile] file"""
 def main():
     flag = sys.argv[1]
     file_name = sys.argv[2]
-    with open(file_name, 'r') as f:
-        source = f.read()
+    try:
+        with open(file_name, 'r') as f:
+            source = f.read()
+    except IOError as e:
+        print "can't open file {}: {}".format(file_name, e.strerror)
+        sys.exit(1)
```

This should return the right error from the errno module.